### ISSUE TITLE
hideScrollButtons not working?

### ISSUE THREAD

**Issue by thatsmekaustubh** [PolymerElements/paper-tabs#100]
<paper-tabs id="scrollableTabs" selected="0" scrollable hideScrollButtons="true">
 <paper-tabs id="scrollableTabs" selected="0" scrollable hideScrollButtons>

Still shows right and left buttons. Please correct me if I am wrong

**Comment by MeTaNoV** [comment]
As an attribute, you should write:
```
<paper-tabs id="scrollableTabs" selected="0" scrollable hide-scroll-buttons>
```

**Comment by thatsmekaustubh** [comment]
Sorry to say it didnt worked as expected. The arrows are still visible

**Comment by bicknellr** [comment]
@thatsmekaustubh I haven't been able to repro using [this JSBin](http://jsbin.com/levavecuge) (latest Chrome / Safari / Firefox on OS X), can you?

**Comment by thatsmekaustubh** [comment]
URL to c9.io prj : https://ide.c9.io/killerj/polymer/
Path to exact file: mockups/mobil/index.html

I believe to edit it you may require permissions

**Comment by bicknellr** [comment]
@thatsmekaustubh Yeah, sign-in seems to be required, could you put (some version of) your code somewhere public?

**Comment by HoraceKeung** [comment]
`hide-scroll-buttons` actually works.
